scripts/st2bootstrap-el8.sh

```diff
@@ -289,6 +289,7 @@ configure_st2_cli_config() {
   # Configure CLI config (write credentials for the root user and user which ran the script)
   ROOT_USER="root"
   CURRENT_USER=$(whoami)
+  CURRENT_GROUP=$(id -gn)
```

Comment:
Was the issue only on EL8 platform or do we need to apply the same patch for other OS systems?

Reply:
@armab The issue I had was installing on a box that had LDAP configured and my user group is 'domain users' since the script just assumed the group matched the user so the chown command fails and the script stops executing. I can avoid the issue by 'sudo su' but I realized that if I get the user's group with 'id -gn' then there is no assumption of what that value should be. This would likely be relevant on the other scripts.

@nmaludy I am reading up on how to perform the steps you requested. I am very new to development so it may take me a bit before I can update. please standby.

Reply:
Thanks for more detailed context, that's helpful! :+1: